layers/+lang/haskell/README.org

```diff
@@ -325,6 +364,12 @@ Flycheck can fail silently for miscellaneous reasons. See the [[#faq][FAQ]]
 for troubleshooting.
```

Comment:
We should remove the section about `flymake` this is horribly outdated.
In addition if we keep this troubleshooting subject, we should move it to the FAQ

Reply:
Agreed, I didn't know much about flymake so I didn't want to touch it at the moment. Troubleshooting part -> it does not seem to bring any new information so I will just remove it.